users/views.py

```diff
@@ -54,5 +54,28 @@ def post(self, request):
 
             return JsonResponse({"message":"success","access_token": access_token}, status=200)
 
+        except KeyError:
+            return JsonResponse({"message": "KEY_ERROR"}, status=400)
+
+class KakaoSigninView(View):
+    def get(self, request):
+        try:
+            access_token     = request.headers.get('Authorization')
+            headers          = {'Authorization': f'Bearer {access_token}'}
+            user_info        = requests.get('https://kapi.kakao.com/v2/user/me', headers=headers).json()                
```

Comment:
`info`라는 말을 붙이는 습관은 좋지 않습니다~

```suggestion
            kakao_user        = requests.get('https://kapi.kakao.com/v2/user/me', headers=headers).json()                
```

Reply:
넵 수정했습니다.